functions/handler.py

```diff
@@ -4,60 +4,47 @@
 import os
 
 REGION = os.getenv('REGION', 'us-west-2')
-service = os.getenv('efServiceName', 'default')
+service = os.getenv('SERVICE', 'default')
 sqs = boto3.client('sqs', region_name=REGION)
 ssm = boto3.client('ssm', region_name=REGION)
 logger = logging.getLogger()
 logger.setLevel(logging.INFO)
 
 
 def lambda_handler(event, context):
-    # This parsing can and will be improved in another PR
-    if 'requestBody' in event['body']:
-        returnDict = dict()
-        returnDict['details'] = {}
-        event = json.loads(event['body'])
-        event_body = event['requestBody']
-        if 'event' not in event_body or 'payload' not in event_body:
-            return {
+    # print(json.dumps(event))
+    # This parsing can and will be improved in the future
+    logger.info("{} service initialized.".format(service))
+    aws_event = event
+    returnDict = dict()
+    returnDict['details'] = {}
+
+    # See if submitted POST body is a valid JSON
+    try:
+        request_body = json.loads(aws_event['body'])
```

Comment:
I don't think aws_event accurately describes this, api_event might be better suited.

Reply:
Sure, I can change it to that.

Just to make sure, the `aws_event` or `api_event` we are referring here is going to be something like this: https://serverless.com/framework/docs/providers/aws/events/apigateway#example-lambda-proxy-event-default and it's different than the external events we would like to consume. The external events (stuff that's submitted to this endpoint) is in the `event` key of that JSON/dictionary.

Reply:
It's not an aws event was all I was trying to point out, it's an api event, regardless of all the fluff the api-gateway tacks onto the accept headers. Everything in the RequestBody is what we are dealing with and interested in, not much else outside of that other than making sure it is sourced from the place we expect it to be, and authenticated properly, which should be vallidated by the authentication piece before it ever makes it this far.